java/src/main/java/ai/rapids/cudf/GroupByResult.java

```diff
@@ -0,0 +1,50 @@
+/*
+ * Copyright (c) 2022, NVIDIA CORPORATION.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package ai.rapids.cudf;
+
+/**
+ * Used by JNI
+ * Used to save groups and keys for `contiguousSplitGroupsAndGenUniqKeys`
+ * Each row in uniq keys table is corresponding to a group
```

Comment:
```suggestion
 * An additional table is introduced to store the group keys, each is corresponding to a group.
```

Reply:
Done

Reply:
But I still saw it here https://github.com/rapidsai/cudf/pull/11614/files#diff-47f36c2be0680602829056353c03d1ada3c396775c81d157c85c3a52876ac644R20